src/sdg/core/management/utils.py

```diff
@@ -165,6 +168,14 @@ def load_upn(data: List[Dict[str, Any]]) -> int:
         if created:
             count += 1
 
+    all_upns = UniformeProductnaam.objects.all()
+
+    deleted_upns = [x for x in all_upns if x not in upn_updated_list]
```

Comment:
Always try to use meaningful variable names.

```suggestion
    deleted_upns = [upn for upn in all_upns if x not in upn_updated_list]
```